gap/examples.gi

```diff
@@ -370,3 +370,48 @@ GeneralisedPetersenGraphCons);
 
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
+
+InstallMethod(KingsGraphCons,
+"for IsMutableDigraph and two positive integers",
+[IsMutableDigraph, IsPosInt, IsPosInt],
+function(filt, n, k)
+  local D, i, j;
+  D := EmptyDigraph(filt, n * k);
+  for i in [1 .. k] do
+    for j in [1 .. (n - 1)] do
+      DigraphAddEdge(D, [((i - 1) * n) + j, ((i - 1) * n) + j + 1]);
+    od;
+  od;
+  for i in [1 .. (k - 1)] do
+    for j in [1 .. n] do
+      DigraphAddEdge(D, [((i - 1) * n) + j, ((i - 1) * n) + j + n]);
+    od;
+  od;
+  for i in [1 .. (k - 1)] do
+    for j in [1 .. (n - 1)] do
+      DigraphAddEdge(D, [((i - 1) * n) + j, ((i - 1) * n) + j + (n + 1)]);
+      DigraphAddEdge(D, [((i - 1) * n) + j + 1, ((i - 1) * n) + j + n]);
+    od;
+  od;
+  return DigraphSymmetricClosure(D);
+end);
+
+InstallMethod(KingsGraphCons,
+"for IsImmutableDigraph and two positive integers",
+[IsImmutableDigraph, IsPosInt, IsPosInt],
+function(filt, n, k)
+  local D;
+  D := MakeImmutable(KingsGraphCons(IsMutableDigraph, n, k));
+  SetIsMultiDigraph(D, false);
+  SetIsSymmetricDigraph(D, true);
+  SetIsConnectedDigraph(D, true);
```

Comment:
I'm pretty sure the same properties should apply as apply to the triangular grid graph. So please add all of those missing ones.

Reply:
Ok, will do.